Backendless/Transaction/Payload/Relation.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
```

Comment:
Let's remove this dependency. Why is it needed?

Reply:
@markpiller It need for JsonIgnore annotations